bot/orchestration/src/main/kotlin/ai/tock/bot/orchestration/bot/primary/DefaultPrimaryBotOrchestrationEventHandler.kt

```diff
@@ -0,0 +1,30 @@
+package ai.tock.bot.orchestration.bot.primary
+
+import ai.tock.bot.definition.StoryDefinition
+import ai.tock.bot.engine.BotBus
+import ai.tock.bot.orchestration.shared.AvailableOrchestrationResponse
+
+open class DefaultPrimaryBotOrchestrationEventHandler(private val comebackStory: StoryDefinition) : PrimaryBotOrchestrationEventHandler {
+    override fun onStarOrchestration(bus: BotBus, orchestrationResponse: AvailableOrchestrationResponse) {
+        bus.send("Pour vous répondre je passe la main à mon collègue ${orchestrationResponse.targetBot.botLabel} !")
```

Comment:
@vsct-jburet  do we want to keep this french sentence ?

Reply:
no !:)